client/beefy/src/lib.rs

```diff
@@ -40,6 +40,12 @@ mod worker;
 
 pub mod notification;
 
+/// payload type.
+pub type Payload = Vec<(BeefyPayloadId, Vec<u8>)>;
```

Comment:
IMHO we shouldn't be exposing the `Vec` directly. I'd suggest to wrap it into a struct and provide a bunch of utility methods to get/add specific payloads.

Something along the lines:
```rust
#[derive(Decode, Encode, Debug, PartialEq, Eq, Clone)]
pub struct Payload {
  payload: Vec<(BeefyPayloadId, Vec<u8>)>,
}

impl Payload {
  /// ...
  ///
  /// Returns a raw payload under given `id`. If the `BeefyPayloadId` is not found in the payload `None` is returned.
  pub fn get_raw(&self, id: &BeefyPayloadId) -> Option<T> {
   ...
  }
  /// ...
  ///
  /// Returns a decoded value `T`, in case the value is not there or the encoding does not match it returns `None`.
  pub fn get_decoded<T: Decode>(&self, id: &BeefyPayloadId) -> Option<T> {
     ...
  }  
  /// ...
  pub fn push_raw(&mut self, id: BeefyPayloadId, value: Vec<u8>) {
      // push into the right position.
  }
}

```